src/main/java/edu/pav/PatientTrackerSystem/controller/DoctorController.java

```diff
@@ -77,6 +79,33 @@ private Boolean existsDoctor(String email) {
         return doctorRepository.findByEmail(email) != null;
     }
 
+    private String encryptString(String password) {
```

Comment:
Should we move this to utils ni future commit?

Reply:
Yes